phishingpage.py

```diff
@@ -0,0 +1,153 @@
+"""
+This module was made to handle all the phishing related operations for
+Wifiphisher.py
+"""
+
+import urllib
+import os
+
+# set of dictionaries to store URLs and names
+LINKSYS = {"index.html": "http://pastebin.com/raw.php?i=b0Uz1sta",
+           "Linksys_logo.png": "https://i.imgur.com/slBTPcu.png"}
+TEMPLATE_TO_URL = {"linksys": LINKSYS, "minimal": None,
+                   "connection_reset": None, "office365": None}
+
+PHISHING_PAGES_DIR = "phishing-pages/"
+
+
+class UrlNotAvailable(Exception):
+    """ Exception class to raise in case of a invalid URL. """
+    def __init__(self):
+        Exception.__init__(self, "The given URL is not available!")
```

Comment:
This is not always the case. Sometimes the user doesn't have Internet access. I propose changing the error message to "Could not reach the given URL!".